host/class/uvc/usb_host_uvc/uvc_control.c

```diff
@@ -16,6 +17,8 @@
 #include "uvc_descriptors_priv.h"
 #include "uvc_check_priv.h"
 
+#define FLOAT_EQUAL(a, b) (fabs(a - b) < 0.0001f) // For comparing float values with acceptable difference (epsilon value)
```

Comment:
Here, you are using math function for `double` type, not for `float`
```suggestion
#define FLOAT_EQUAL(a, b) (fabsf(a - b) < 0.0001f) // For comparing float values with acceptable difference (epsilon value)
```